app/components/rate-goal/animations/cactus.jsx

```diff
@@ -32,6 +32,8 @@ const createAnimation = ({target, options, }) => {
     { visibility: 'visible', },
   ];
 
+<<<<<<< HEAD
```

Comment:
something went wrong with pushing to github i think, it was ok on my local branch. weird.